lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -188,36 +189,35 @@ function convertMessageToCtc(lhlMessage, examples = {}) {
  * @param {string} lhlMessage
  */
 function _lhlValidityChecks(lhlMessage) {
-  let parsedMessage;
+  let parsedMessageElements;
   try {
-    parsedMessage = MessageParser.parse(lhlMessage);
+    parsedMessageElements = MessageParser.parse(lhlMessage);
   } catch (err) {
     if (err.name !== 'SyntaxError') throw err;
-    // Improve the intl-messageformat-parser syntax error output.
-    /** @type {Array<{text: string}>} */
-    const expected = err.expected;
```

Comment:
this doesn't exist any more.

Reply:
There was a "location" field when being/end markers, could use that to point to the specific characters that are problematic.

Can't promise fancy ascii pointers like you'd see elsewhere though :P